fv/datastore/datastore_describe.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2018 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package datastore
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+)
+
+type DatastoreType int
+
+const (
+	DatastoreEtcdV3 DatastoreType = 1 << iota
+	DatastoreK8s
+
+	DatastoreAll = DatastoreEtcdV3 | DatastoreK8s
+)
```

Comment:
Those are strings, so you'd need a different way of representing this (`DatastoreAll`), but I guess that wouldn't be hard.